v3/mqtt.go

```diff
@@ -190,9 +190,6 @@ type (
 		resultCh chan<- MqttResponse
 	}
 
-	// Type alias for something that we use everywhere
-	mqttSendPacketChan chan packetSendData
```

Comment:
@shigmas this looks unused. Can I delete this?

Reply:
Yeah, looks like it. I was cleaning things earlier, and I must have forgotten to remove this.